src/python/pants/bin/remote_pants_runner.py

```diff
@@ -63,8 +185,26 @@ def __init__(self, exiter, args, env, bootstrap_options, stdin=None, stdout=None
   def _trapped_signals(self, client):
     """A contextmanager that overrides the SIGINT (control-c) and SIGQUIT (control-\) handlers
     and handles them remotely."""
+    # TODO: if this control-c is done at the wrong time (before the pantsd-runner process begins and
```

Comment:
lets extract all the ctrl-c changes from this PR into a separate one.

Reply:
This file has been reverted entirely!